framework/src/main/java/org/checkerframework/common/accumulation/AccumulationTransfer.java

```diff
@@ -22,7 +22,7 @@
  * <p>Subclasses should call the {@link #accumulate(Node, TransferResult, String...)} accumulate}
  * method to add a string to the estimate at a particular program point.
  */
-public class AccumulationTransfer extends CFTransfer {
+public abstract class AccumulationTransfer extends CFTransfer {
```

Comment:
I don't *object* to this change, but I don't understand the reasoning for making it in this PR.